src/test/java/pet/odyvanck/petclinic/data/OwnerTestFactory.java

```diff
@@ -0,0 +1,172 @@
+package pet.odyvanck.petclinic.data;
+
+
+import pet.odyvanck.petclinic.domain.Owner;
+import pet.odyvanck.petclinic.domain.User;
+import pet.odyvanck.petclinic.web.dto.owner.OwnerCreationRequest;
+import pet.odyvanck.petclinic.web.dto.owner.OwnerResponse;
+import pet.odyvanck.petclinic.web.dto.owner.OwnerUpdateRequest;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Factory class providing ready-to-use test objects for Owner-related tests.
+ */
+public final class OwnerTestFactory {
+
+    private OwnerTestFactory() {
+    }
+
+    public static OwnerCreationRequest createOwnerCreationRequest() {
+        return new OwnerCreationRequest(
+                "John",
```

Comment:
хорошей практикой является для строковых тестовых данных использовать в качестве значения названия поля. Так при написание тестов можно быстрее найти какие то проблемы 
```suggestion
                return new OwnerCreationRequest(
                "firstName",
                "lastName",
                "password",
                "+1234567890",
                "email@mail.com",
                "address"
        );
```